packages/utils/src/lib/reports/generate-md-report-categoy-section.ts

```diff
@@ -13,112 +13,103 @@ import {
   scoreMarker,
 } from './utils';
 
-const { link, section, h2, lines, li, bold: boldMd, h3, indentation } = md;
-
 export function categoriesOverviewSection(
   report: Pick<ScoredReport, 'categories' | 'plugins'>,
-): string {
+): MarkdownDocument | null {
```

Comment:
This function never returns `null`. 🤨 
```suggestion
): MarkdownDocument {
```